junit-jupiter-api/src/main/java/org/junit/jupiter/api/AssertionUtils.java

```diff
@@ -56,6 +56,15 @@ static String nullSafeGet(Supplier<String> messageSupplier) {
 		return (messageSupplier != null ? messageSupplier.get() : null);
 	}
 
+	static String nullSafeGet(Object messageContainer) {
+		// messageContainer suppose to contain a message string or a message supplier
+		// added to avoid the need to wrap message into lambda (() -> message)
+		return (messageContainer != null
+				? (messageContainer instanceof Supplier ? nullSafeGet(((Supplier<?>) messageContainer).get())
+						: messageContainer.toString())
+				: null);
+	}
```

Comment:
@sbrannen  My previous comment lost. More than 11 methods need to be copy/pasted without this method with `Object` parameter. Except of this method `Object messageContainer` is present only in private methods.